dpkg/scanner_test.go

```diff
@@ -825,6 +825,41 @@ func TestScanner(t *testing.T) {
 	}
 }
 
+func TestScannerACMEImage(t *testing.T) {
```

Comment:
This test needs to actually fetch the relevant layer or craft a layer (like in `TestExtraMetadata`), checking in the layer won't work. Also, this name doesn't explain what it's actually testing.